migrations/versions/d63493af0a71_create_meteorological_wave_data.py

```diff
@@ -0,0 +1,92 @@
+"""create meteorological_data and wave_data tables
+
+Revision ID: d63493af0a71
+Revises: 6fc1a99cefd5
+Create Date: 2022-08-20 14:01:00.212730
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "d63493af0a71"
+down_revision = "6fc1a99cefd5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "meteorological_data",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("station_id", sa.String(length=10), nullable=False),
+        sa.Column("date_recorded", sa.DateTime(), nullable=False),
+        sa.Column("wind_direction", sa.Integer(), nullable=True),
+        sa.Column("wind_speed", sa.Float(), nullable=True),
+        sa.Column("wind_gust", sa.Float(), nullable=True),
+        sa.Column("wave_height", sa.Float(), nullable=True),
+        sa.Column("dominant_wave_period", sa.Float(), nullable=True),
+        sa.Column("average_wave_period", sa.Float(), nullable=True),
+        sa.Column("wave_direction", sa.Integer(), nullable=True),
+        sa.Column("sea_level_pressure", sa.Float(), nullable=True),
+        sa.Column("pressure_tendency", sa.Float(), nullable=True),
+        sa.Column("air_temperature", sa.Float(), nullable=True),
+        sa.Column("water_temperature", sa.Float(), nullable=True),
+        sa.Column("dewpoint_temperature", sa.Float(), nullable=True),
+        sa.Column("visibility", sa.Float(), nullable=True),
+        sa.Column("tide", sa.Float(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["station_id"],
+            ["buoys.station_id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_meteorological_data_id"), "meteorological_data", ["id"], unique=False
+    )
+    op.create_index(
+        op.f("ix_meteorological_data_station_id"),
+        "meteorological_data",
+        ["station_id"],
+        unique=False,
+    )
+    op.create_table(
+        "wave_data",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("station_id", sa.String(length=10), nullable=False),
+        sa.Column("date_recorded", sa.DateTime(), nullable=False),
+        sa.Column("significant_wave_height", sa.Float(), nullable=True),
+        sa.Column("swell_height", sa.Float(), nullable=True),
+        sa.Column("swell_period", sa.Float(), nullable=True),
+        sa.Column("wind_wave_height", sa.Float(), nullable=True),
+        sa.Column("wind_wave_period", sa.Float(), nullable=True),
+        sa.Column("swell_direction", sa.String(), nullable=True),
+        sa.Column("wind_wave_direction", sa.String(), nullable=True),
+        sa.Column("steepness", sa.String(), nullable=True),
+        sa.Column("average_wave_period", sa.Float(), nullable=True),
+        sa.Column("mean_wave_direction", sa.Integer(), nullable=True),
```

Comment:
FYI, I called this column "dominant_wave_direction" in `pybuoy` v0.4.0

Reply:
Why dominant? I understood the definition to be as average wave direction.

Reply:
> Why dominant? I understood the definition to be as average wave direction.

I based it off [NDBC documentation of measurement descriptions](https://www.ndbc.noaa.gov/measdes.shtml).

Here's a snippet that's in reference to the last column ("MWD"):
> The direction from which the waves at the dominant period (DPD) are  		coming. The units are degrees from true North, increasing clockwise, with North as 0 (zero) degrees and East as 90 degrees.

Because they reference the dominant period in the description, I decided to adjust the name accordingly to reflect that.

Why did you call it mean wave direction?

Reply:
Well the column is abbreviated as MWD. Their definition isn't also saying it's the dominant wave direction, it's saying it's the average direction of the waves during the dominant period. I called it mean to also be consistent their abbreviations. Think about it, if they're showing MWD and then pybuoy is showing Dominant_Wave_Direction, their will be confusion about where did they get the Dominant part from in the MWD column. That's why I marked it as mean.

Reply:
> it's saying it's the average direction of the waves during the dominant period

Where is it saying that? I can't find it – maybe I missed it.

Reply:
Oh no dude... I don't think it even means mean... I think it means Maximum Wave Direction 

> Dominant wave period (seconds) is the period with the maximum wave energy

And if MWD is meant to represent the direction of the waves during the DPD which is the wave with the maximum wave energy in a given time period... then wouldn't that be max wave direction? Maybe these guys just suck at giving things labels. I'm about to email them asking what MWD actually stands for.

Reply:
Great idea. Let me know what they say!